test/restart/mzcompose.py

```diff
@@ -164,8 +164,68 @@ def workflow_storage_managed_collections(c: Composition) -> None:
         raise Exception("user shards empty or not equal after restart")
 
 
+def workflow_allowed_cluster_replica_sizes(c: Composition) -> None:
+    c.up("testdrive", persistent=True)
+    c.start_and_wait_for_tcp(services=["materialized"])
+
+    c.testdrive(
+        "\n".join(
```

Comment:
Not blocking, but we usually use dedent with """ to write multi-line testdrive fragments in Python. Or create test/restart/allowed-cluster-replica-sizes/before.td and test/restart/allowed-cluster-replica-sizes/after.td.